### ISSUE TITLE
Team review of slides

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/reproducible-research#87]
We now have "completed" slide decks, in [this](https://drive.google.com/drive/folders/1z9dvfD2RHDky-OH_nDBQaZhpHQ8Q5NcE?usp=sharing) folder in the CCDL Google Drive. 

Data team @jaclyn-taroni @cbethell @allyhawkins @Jen-OMalley @envest  - please feel free to leave comments and feedback directly in slides (or in this issue!) **no later than Wednesday 4 PM.** *This deadline is CRUCIAL (and sooner the better!) so we can incorporate feedback!* If you have no feedback, let us know here when you've reviewed and are all set. 

Once slides have been reviewed by the team and feedback has been incorporated, we can proceed to publish slide links in the schedule (#73) and release a full PDF of all slides (#34) to includes in `docs/`.

**Comment by allyhawkins** [comment]
I went through all the slides and left a few comments with some minor opinions, but overall just wanted to say that the content and slides look awesome 🎉 This is going to be a really great workshop!

**Comment by jaclyn-taroni** [comment]
I've taken a look and added a few comments. Any minor formatting issues (errant spaces, etc.) I noticed I just went ahead and fixed. 

I learned a bunch about GitKraken I did not know. I also think I would have benefited very much from this material as a graduate student! 🎉

**Comment by envest** [comment]
Wow! Comments made. Great work, content producers!

**Comment by Jen-OMalley** [comment]
Heads up that I added an ScPCA slide to the [**workshop intro**](https://docs.google.com/presentation/d/1GgqWl6las6ekQtKLmPzJ8UD_cftIbJjDTS2gJpStcMQ/edit#slide=id.p). I also added Deepa to the slides. Participants will meet her at dinner! 

ScPCA is currently slide 9, right after the staff intros. It includes content about usability testing so it seemed to make sense to place it right after Deepa's introduction. Tagging **@jashapiro @sjspielman @dvenprasad**, but anyone feel free to review and let me know if you feel the slide belongs elsewhere. (Or go ahead and move it as you see fit!)
